src/odemis/gui/comp/overlay/ellipse.py

```diff
@@ -78,37 +114,47 @@ def on_left_up(self, evt):
 
             # Set the points VA after drawing because draw() gathers the points
             self.cnvs.update_drawing()
-            if self.selected.value:
+            # The rotation point is outside the shape and cannot be captured by selection VA
+            # Update the points VA if the selection mode is SEL_MODE_ROTATION
+            if self.selected.value or is_rotation:
                 self.points.value = self._points
         WorldOverlay.on_left_up(self, evt)
 
-    def draw(self, ctx, shift=(0, 0), scale=1.0):
+    def draw(self, ctx, shift=(0, 0), scale=1.0, dash=False):
```

Comment:
Every other `draw` function with a `dash` argument have it default to `dash=True`. Please do the same here.
While at it, you can indicate the type is a `bool`.